### PR TITLE
Fix x.509 configuration requirements for RACF

### PR DESCRIPTION
**Pull request by pablocarle**
Describe your pull request here:

Testing showed the documentation was missing a few important topics regarding RACF configuration for mapping certificate distinguished names to a user.

List the file(s) included in this PR:
- docs/user-guide/authenticating-with-client-certificates.md
- versioned_docs/version-v2.18.x/user-guide/authenticating-with-client-certificates.md

### CODE REVIEW

File: docs/user-guide/authenticating-with-client-certificates.md [hunk 1]
```diff
@@ -30,36 +31,54 @@ When sending a request to the login endpoint with a client certificate, the Gate
...
+
+Register the client certificate with the user IDs in your ESM. 
 
-Register the client certificate with the user ID in your ESM. The following commands apply to both the internal API ML mapper and ZSS.
+The following commands show options for both the internal API ML mapper and ZSS.
 
-**RACF** 
+:::note
+
+If using the internal API ML mapper (default from Zowe v3) and the MAP / CERTMAP option with distinguished name filters, use the `CHCKCERT` or equivalent command on the certificate to use the same order and format as displayed.
```
Comment (richard-salac) [inline docs/user-guide/authenticating-with-client-certificates.md:55]: same order of DN fields

File: docs/user-guide/authenticating-with-client-certificates.md [hunk 2]
```diff
@@ -30,36 +31,54 @@ When sending a request to the login endpoint with a client certificate, the Gate
...
   SDNFILTER('<subject's-distinguished-name-filter>')
   WITHLABEL('<label>')
+  SETROPTS RACLIST(DIGTMAP) REFRESH
+  ```
+
+  Alternatively, if you disabled the internal API ML mapper, use the following command to add the certificate to an ACID
+
+  ```racf
+  RACDCERT ADD(<dataset>) ID(<userid>) WITHLABEL('<label>') TRUST
+  SETROPTS RACLIST(DIGTCERT, DIGTRING) REFRESH
```
[the remark this answers is not in the document]
Comment (pablocarle) [reply docs/user-guide/authenticating-with-client-certificates.md:81]: I suppose we assume those to be enabled already, certificates and keyrings are more common than mappings (either by distinguished names or OIDC)

[the remark this answers is not in the document]
Comment (pablocarle) [reply docs/user-guide/authenticating-with-client-certificates.md:81]: Anyway, we are currently working on an installation in a vanilla environment, this is something we can evaluate there.

File: docs/user-guide/authenticating-with-client-certificates.md (hunk 1 above)
Comment (pablocarle) [reply docs/user-guide/authenticating-with-client-certificates.md:55]: I'll add this, thank you

**Review by janan07 — APPROVED** [review]
Reviewed and edited by doc squad

File: docs/user-guide/authenticating-with-client-certificates.md [hunk 3]
```diff
@@ -69,59 +88,67 @@ Alternatively, if you are using the internal API ML mapper, use the following co
...
-To validate that the client certificate functionality works properly, call the login endpoint with the certificate that was set up using the steps in [Configure your z/OS system to support client certificate authentication for a specific user](#configure-your-zos-system-to-support-client-certificate-authentication-for-a-specific-user) described previously in this article. 
+To validate that the client certificate functionality works properly, call the login endpoint with the certificate that was set up using the steps in [Configure your z/OS system to support client certificate authentication for a specific user](#configure-your-zos-system-to-support-client-certificate-authentication-for-specific-users) described previously in this article.
 
 Validate using _CURL_, a command line utility that runs on Linux based systems:
 
 **Example:**
-```
+
+```bash
 curl --cert /path/to/cert.pem --key /path/to/key.pem https://api-mediation-layer:7554/gateway/api/v1/login
```
[the remark this answers is not in the document]
Comment (pablocarle) [reply docs/user-guide/authenticating-with-client-certificates.md:151]: I think you are right, the URL is wrong

[the remark this answers is not in the document]
Comment (pablocarle) [reply docs/user-guide/authenticating-with-client-certificates.md:151]: Thanks for the find

**Review by richard-salac — APPROVED** [review]

**Pull request merged**